main/docs/customize/forms/flows/integrations/xml.mdx

```diff
@@ -0,0 +1,131 @@
+---
+'og:description': Learn how to configure your XML flow actions to convert JSON to
+  XML and XML to JSON.
+'og:image': https://cdn2.auth0.com/docs/1.14553.0/img/share-image.png
+'og:title': XML
+'og:url': https://auth0.com/docs/
+permalink: xml
+title: XML
+'twitter:description': Learn how to configure your XML flow actions to convert JSON
+  to XML and XML to JSON.
+'twitter:title': XML
+---
+This list of XML actions allows you to convert JSON objects to XML string and XML string to JSON objects.
+
+## Convert JSON to XML
+
+Converts a JSON object to an XML string.
+
+<Frame>![](https://images.ctfassets.net/cdy7uua7fh8z/3GW2EhBMIX3YDPaqc3rXrZ/7706a863700df51376d645a7d9292a27/convertjsontoxml.png)</Frame>

```

Comment:
We need to self host all the assets currently hosted on `https://images.ctfassets.net`.